Source/ACE/Entity/Scroll.cs

```diff
@@ -0,0 +1,157 @@
+﻿// WeenieType.Scroll
+
+using ACE.DatLoader.FileTypes;
+using ACE.Entity.Actions;
+using ACE.Entity.Enum;
+using ACE.Entity.Enum.Properties;
+using ACE.Network;
+using ACE.Network.GameEvent.Events;
+using ACE.Network.GameMessages.Messages;
+using ACE.Network.Motion;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace ACE.Entity
+{
+    public class Scroll : WorldObject
+    {
+        private static readonly UniversalMotion motionReading = new UniversalMotion(MotionStance.Standing, new MotionItem(MotionCommand.Reading));
+        private static readonly UniversalMotion motionReady = new UniversalMotion(MotionStance.Standing, new MotionItem(MotionCommand.Ready));
+
+        public Scroll(AceObject aceObject)
+            : base(aceObject)
+        {
+            SpellTable table = SpellTable.ReadFromDat();
+
+            Use = $"Inscribed spell: {table.Spells[SpellId].Name}\n";
+            Use += $"{table.Spells[SpellId].Desc}";
+
+            LongDesc = "Use this item to attempt to learn its spell.";
+
+            Power = table.Spells[SpellId].Power;
+            School = table.Spells[SpellId].School;
+        }
+
+        private uint SpellId
+        {
+            get { return (uint)Spell.Value; }
+        }
+
+        private uint Power
```

Comment:
As I understand things, this is more accurately described as "MinimumSkillLevel".  I'd prefer a rename, but at least document it.